src/components/site/SideBarSection.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const SidebarSection = props => {
+  const { title, children } = props;
+  return (
+    <div className="dg_sidebar">
```

Comment:
I know this is a rename thing, but it probably makes sense to name teh class based on the componet. In this case

`dg_sidebar__section`

`dg_sidebar` sounds like it's container, which I think we will maybe need.

Reply:
Well not really. That div IS the entire sidebar with sections. Maybe we should wrap the h2 in a div with a .dg_sidebar__section class since that is an actual "section header"? Would then look something like this which is probably better. 

![image](https://user-images.githubusercontent.com/40430541/67953333-4833e380-fbc5-11e9-84ef-4bece526cc8a.png)

Reply:
This has been done

Reply:
thanks @tmccoy529 this makes a lot of sense to me.